src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -59,6 +59,10 @@
 BASE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
                          "test_files", "dynamo0p3")
 
+# Our configuration object
+from psyclone.configuration import ConfigFactory
+_CONFIG = ConfigFactory().create()
```

Comment:
Again should this be created in each test so tests don't interfere?. If not then leave as it is. Especially as it i used alot!